Dockerfile-operator

```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM confluentinc/cp-kafka-connect:5.3.1
+ARG CP_VERSION
 
-ENV CONNECT_PLUGIN_PATH="/usr/share/java,/usr/share/confluent-hub-components"
+FROM confluentinc/cp-server-connect-operator:${CP_VERSION}.0
```

Comment:
Also, please add a comment explaining why `.0`.

Reply:
I'll do some research / ask engineering for details